packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.stories.tsx

```diff
@@ -27,9 +26,22 @@ const wait = (delay: number) =>
 
 function init() {
   storiesOf('routes/ProductV2/SubscriptionCreate', module)
-    .add('default', () => <Subject />)
+    .add('default', () => (
+      <Subject
+        transactionInProgress={false}
+        setTransactionInProgress={() => {}}
```

Comment:
Aren't these the defaults?  Why pass them so many times?

Reply:
Removed excess arguements